pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -344,6 +378,8 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		{
 			ObjectMeta: kapi.ObjectMeta{
 				Name: ViewRoleName,
+				Annotations: map[string]string{
+					oapi.OpenShiftDescription: "A user who cannot make any modifications, but can see most objects in a project. They cannot view or modify roles or bindings or secrets."},
```

Comment:
A user who can view but not edit any resources within the project. They can not view secrets.